include/storage/flash_map.h

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ * Copyright (c) 2015 Runtime Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Public API for flash map
+ */
+
+#ifndef ZEPHYR_INCLUDE_FLASH_MAP_H_
```

Comment:
Looks like this commit also has issues with include guards.... does this actually work? It looks broken to me.

This file is included from include/flash_map.h, which has already defined ZEPHYR_INCLUDE_FLASH_MAP_H_.

Reply:
this is broken indeed, if you use the old location, in the tree however we removed usage of old location, so all works. Need to test allowing deprecation messages. Fixed now.

Reply:
Thanks